change/office-ui-fabric-react-2019-11-25-18-02-32-user-v-satgar-ExposePropForDialog.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

This should be patch now
```suggestion
  "type": "patch",
```

---

done